percy/providers/app_automate.py

```diff
@@ -30,6 +31,22 @@ def set_debug_url(self, session_details):
         session_hash = session_details.get("sessionHash")
         self.debug_url = "https://app-automate.browserstack.com/dashboard/v2/builds/" + build_hash + "/sessions/" + session_hash
 
+    def _get_tiles(self, **kwargs):
+        fullpage_ss = kwargs.get('fullpage_screenshot', False)
+        if fullpage_ss:
+            tile_count = kwargs.get('num_of_tiles', 4)
+            data = self.execute_percy_screenshot(self.metadata.device_screen_size.get('height', 1), tile_count, self.metadata.scale_factor)
+            tiles = []
+            status_bar_height = self.metadata.status_bar_height
+            nav_bar_height = self.metadata.navigation_bar_height
+
+            for tile in data.get('result'):
```

Comment:
```suggestion
            for tile_data in data.get('result'):
```
maybe that as its not a tile object yet.